src/broker/broker.c

```diff
@@ -781,7 +781,7 @@ receiver_thr_f (void *arg)
   int job_count;
   int read_len;
   int one = 1;
-  char cas_req_header[SRV_CON_CLIENT_INFO_SIZE];
+  char cas_req_header[SRV_CON_CLIENT_INFO_SIZE + 4];
```

Comment:
If 'STATUS' flag is changed to 'ST', the cas_req_header size does not need to be changed.

Reply:
OK, your point might be good.
I will modify that JDBC sends just 10 bytes with "ST" + pid + session-id.